doc/man1/openssl-verification-options.pod

```diff
@@ -548,10 +567,14 @@ This option has no effect and is retained for backward compatibility only.
 
 =head2 Certificate Extensions
 
-Options like B<-purpose> lead to checking the certificate extensions,
+Options like B<-purpose> lead to checking specific certificate extensions,
 which determine what the target certificate and intermediate CA certificates
 can be used for.
 
+The follwing two subsections on the B<basicConstraints> and <keyUsage>
```

Comment:
nit: following

Reply:
Fixed this and the two further nits below.